pkg/controller/bootstrap/bootstrap.go

```diff
@@ -59,9 +61,16 @@ func (b *Bootstrap) Run(destDir string) error {
 		return err
 	}
 
+	scheme := runtime.NewScheme()
+	v1.Install(scheme)
```

Comment:
One thing I found is [the docs here](https://godoc.org/k8s.io/apimachinery/pkg/runtime) say:
> It's recommended that you register your types with runtime in your package's init function.

https://medium.com/@arschles/go-experience-report-generics-in-kubernetes-25da87430301
is also useful on this topic.